lib/components/Topbar/topbar.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:flutter/material.dart';
+import 'package:caderneta_campo_digital/global/global.dart';
+import 'package:mask_text_input_formatter/mask_text_input_formatter.dart';
+
+class Topbar extends StatelessWidget implements PreferredSizeWidget {
+  final Size topbarHeight;
+  const Topbar({Key? key, required this.topbarHeight}) : super(key: key);
+
+  @override
+  Size get preferredSize => topbarHeight;
+
+  @override
+  Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
+    var maskFormatter = MaskTextInputFormatter(mask: "###.###.###-##");
+    String name = SharedInfo.actualUser.name;
+    String cpf = maskFormatter.maskText(SharedInfo.actualUser.cpf);
+   
+    return AppBar(
+      automaticallyImplyLeading: false,
+      toolbarHeight: size.height * 0.14,
+      title: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Text(
+              'Olá, $name',
+              style: TextStyle(
+                color: Colors.white,
+                fontSize: 20,
+                fontFamily: 'Roboto-Bold',
+              ),
+            ),
+            Text(
+              'CPF: $cpf',
+              style: TextStyle(
+                color: Colors.white,
+                fontSize: 20,
+                fontFamily: 'Roboto-Regular',
+              ),
+            ),
+          ],),
+      actions: <Widget>[
+        IconButton(
+          icon: const Icon(Icons.person),
+          iconSize: 50,
+          color: Colors.white,
+          padding: EdgeInsets.all(30),
+          // ignore: no-empty-block
+          onPressed: () {},
+        ),
```

Comment:
A área de clique do botão de perfil está absurdamente grande com esse padding de 30px, sugiro que coloquem o padding como 0, já que o IconButton tem um padding padrão, e adicionem um margin a partir de um Container para esse IconButton.
```suggestion
Container(
    margin: EdgeInsets.only(right: 20),
     child: IconButton(
          icon: const Icon(Icons.person),
          iconSize: 50,
          color: Colors.white,
          padding: EdgeInsets.all(0),
          // ignore: no-empty-block
          onPressed: () {},
        ),
  ),
```